experimental/packages/exporter-logs-otlp-grpc/README.md

```diff
@@ -0,0 +1,96 @@
+# OpenTelemetry Collector Logs Exporter for node with grpc
+
+[![NPM Published Version][npm-img]][npm-url]
+[![Apache License][license-image]][license-image]
+
+**Note: This is an experimental package under active development. New releases may include breaking changes.**
+
+This module provides exporter for node to be used with OTLP (`grpc`) compatible receivers.
+Compatible with [opentelemetry-collector][opentelemetry-collector-url] versions `>=0.75.0`.
+
+## Installation
+
+```bash
+npm install --save @opentelemetry/exporter-logs-otlp-grpc
+```
+
+## Service Name
+
+The OpenTelemetry Collector Exporter does not have a service name configuration.
+In order to set the service name, use the `service.name` resource attribute as prescribed in the [OpenTelemetry Resource Semantic Conventions][semconv-resource-service-name].
+To see documentation and sample code for the traces exporter, as well as instructions for using TLS, visit the [Collector Trace Exporter for web and node][trace-exporter-url].
+To see documentation and sample code for the metric exporter, see the [exporter-metrics-otlp-grpc package][metrics-exporter-url]
+
+## Logs in Node - GRPC
+
+The OTLPLogsExporter in Node expects the URL to only be the hostname. It will not work with `/v1/logs`. All
+options that work with trace also work with logs.
+
+```js
+import {
+  LoggerProvider,
+  SimpleLogRecordProcessor,
```

Comment:
We may want to recommend the BatchLogRecordProcessor here, as people may be inclined to copy the code from this readme and the SimpleLogRecordProcessor may introduce some overhead.

Reply:
FYI, `SimpleSpanProcessor` is used many example packages and README files.

Reply:
`SimpleSpanProcessor` is useful for getting started because you don't have delays in demos and that kind of thing. I think we need to be more consistent as a project in what we recommend for local testing vs production. Either is fine here since we should apply our decision to all packages later anyway.